relays/sonoff_mini_r2/manifest.yml

```diff
@@ -0,0 +1,107 @@
+blueprint_spec: device/1.0
+
+display_name: Sonoff MINI R2
+description: Smart Relay
+
+communication_module:
+  product: ENP-VIRTUAL
+  lua_file: firmware.lua
+
+properties:
+  vendor:
+    type: string
+    display_name: Vendor
+  serial_number:
+    type: string
+    display_name: Serial Number
+  model:
+    type: string
+    display_name: Model
+  fw_version:
+    type: string
+    display_name: Sonoff Firmware Version
+
+telemetry:
+  status:
+    type: string
+    display_name: Switch state
+    enum:
+      - switch_on
+      - switch_off
+      - no_data
+  connection_status:
+    type: string
+    display_name: Connection Status
+    enum:
+      - ok
+      - no_data
+      - error
+  signal:
+    type: integer
+    display_name: Internet Signal Strength
+  fwversion:
+    type: string
+    display_name: Sonoff Firmware Version
+
+alerts:
+  connection_err:
+    severity: error
+    display_name: Connection Error
+    description: >
+      Please use 'Set Up Connection' command to set up your
+      Sonoff device configuration.
+  no_data:
+    severity: error
+    display_name: No data from device
+    description: >
+      Can't read data from Sonoff MINI R2.
+      Please check configuration parameters.
+
+
+command_groups:
+  connection:
+    display_name: Connection
+
+commands:
+  write_configuration:
+    display_name: Set Up Connection
+    description: Set your Sonoff account parameters
+    group: connection
+    populate_values_command: read_configuration
+    ui:
+      icon: file-document-edit-outline
+    arguments:
+      ip_address:
+        display_name: Sonoff IP address
+        type: string
+        required: true
+      port:
+        display_name: Sonoff port
+        type: string
+        required: true
+  read_configuration:
+    display_name: Read Connection Parameters
+    description: Read your Sonoff IP Address and port information
+    group: connection
+    ui:
+      icon: file-check-outline
+  activate_switch:
+    display_name: Activate Switch
```

Comment:
better rewrite to _Control Switch_

Reply:
I think you can rename command itself here and in firmware.lua to `control_switch`